doc/multiple_robot_arms/multiple_robot_arms_tutorial.rst

```diff
@@ -0,0 +1,606 @@
+                                    child(2):  lft_arm_link5_sc
+                                child(2):  lft_arm_link4_sc
+                            child(2):  lft_arm_link3_sc
+                        child(2):  lft_arm_link2_sc
+                    child(2):  lft_arm_link1_sc
+                child(2):  lft_arm_link0_sc
+            child(2):  rgt_arm_link0
+                child(1):  rgt_arm_link1
+                    child(1):  rgt_arm_link2
+                        child(1):  rgt_arm_link3
+                            child(1):  rgt_arm_link4
+                                child(1):  rgt_arm_link5
+                                    child(1):  rgt_arm_link6
+                                        child(1):  rgt_arm_link7
+                                            child(1):  rgt_arm_link8
+                                                child(1):  rgt_arm_hand
+                                                    child(1):  rgt_arm_leftfinger
+                                                    child(2):  rgt_arm_rightfinger
+                                                    child(3):  rgt_arm_hand_sc
+                                                    child(4):  rgt_arm_hand_tcp
+                                            child(2):  rgt_arm_link7_sc
+                                        child(2):  rgt_arm_link6_sc
+                                    child(2):  rgt_arm_link5_sc
+                                child(2):  rgt_arm_link4_sc
+                            child(2):  rgt_arm_link3_sc
+                        child(2):  rgt_arm_link2_sc
+                    child(2):  rgt_arm_link1_sc
+                child(2):  rgt_arm_link0_sc
+
+
+To visually check your multiple robot arm model, ensure that the ``urdf_tutorial`` ROS package is installed and run the command: ::
+
+    roslaunch urdf_tutorial display.launch model:=panda_multiple_arms.urdf
+
+Once Rviz GUI starts, set the fixed frame on the upper left corner to be ``world``. 
+
+.. image:: images/rviz_fixed_frame.png
+   :width: 300pt
+   :align: center
+
+If the previous steps are correctly done, the multiple arms model should show up as follows. 
+
+.. image:: images/rviz_start.png
+   :width: 500pt
+   :align: center
+
+
+This concludes the first step of building the model and verifying it. 
+
+Step 2: Prepare the MoveIt config package using MoveIt Setup Assistant 
+---------------------------------------------------------------------- 
+
+MoveIt Setup Assistant is used to configure our multiple robot arms for using the MoveIt pipeline. 
+
+- Start the MoveIt Setup Assistant: ::
+
+    roslaunch moveit_setup_assistant setup_assistant.launch
+
+Follow the MoveIt Setup Assistant `tutorial <https://ros-planning.github.io/moveit_tutorials/doc/setup_assistant/setup_assistant_tutorial.html>`_ to configure the arms. Note that we will be making a separate move group for each arm and hand. The groups are called ``rgt_arm``, ``lft_arm``, ``rgt_hand``, and ``lft_hand``. 
```

Comment:
As with the previous comment, please change `rgt_hand` to `right_hand` and `lft_hand` to `left_hand`.